### DISCUSSION TITLE
Residual ionospheric phase in corrected ALOS2 interferograms!

### DISCUSSION THREAD

**Discussion by RShamshiri** [isce-framework/isce2#818] [Q&A]
Hello everyone!

I have applied ionospheric correction (using the split-spectrum method) to a couple of ScanSAR (VBD) ALOS2 images using ISCE2 (version 2.5.3). However, I think there might still be some ionospheric phase in the corrected interferograms. For instance, in the example below, I think the artefacts on the right side of the image are residual ionosphere (multi-looking factors are 80 and 448):
![image](https://github.com/isce-framework/isce2/assets/105787277/1318162f-a3b3-4c5d-82e1-8e2e7db80efe)

I still see the residuals when I decrease the multi-looking factors (to 40 and 224):
![image](https://github.com/isce-framework/isce2/assets/105787277/5d485d54-2c83-415d-9b34-39f421587538)

@hfattahi @CunrenLiang I wonder how can I improve the correction.

Kind regards
Roghayeh

**Comment by hfattahi** [comment]
Hi @RShamshiri , Thanks for sharing your results. I don't really see a very evident artifact. I would say the ionospheric correction has worked very nicely here. The residual can be troposphere or other environmental effects. What do you think @CunrenLiang ?

**Reply by CunrenLiang** [reply]
Yes. I think there are no big issues with ionosphere estiamtion. The second one is good enough. The coherence on the right side is low. This leads to poor performance of the ionospehric estimation.

If you want to do more test, you can try these ionosphere estimation parameters here:
https://github.com/isce-framework/isce2/blob/main/examples/input_files/alos2/alos2App.xml#L277-L340

especially:
https://github.com/isce-framework/isce2/blob/main/examples/input_files/alos2/alos2App.xml#L333-L334
https://github.com/isce-framework/isce2/blob/main/examples/input_files/alos2/alos2App.xml#L340
